main/app_verifier_test.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"github.com/stretchr/testify/assert"
+	"io"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+)
+
+//Test_E2E_API is quick and dirty full E2E blackbox integration test program that can be run against
+//either live or stubbed out dependencies.  It can be used in a few different ways:
+//1. During development by engineers as a smoke test
+//2. As part of a CD/CI pipeline with stubbed data
+//3. As a cronned health check against a live production instance
+
+func Test_E2E_API(t *testing.T) {
+	/*
+		Example code to show how Wikipedia fetcher can be stubbed out:
+		if (config.usestub){
+			indexer.Fetcher = func(date time.Time) ([]messages.ArticleCount, error) {
+				// returns stubbed data
+			}
+		}
+	*/
+	go main()
+	time.Sleep(4 * time.Second)
+	//mostviewed HappyPath
+	r, _ := http.Get("http://localhost:8080/mostviewed/20220101/20220102")
+	defer r.Body.Close()
+	bytes, _ := io.ReadAll(r.Body)
+	payloadString := string(bytes[:])
+	assert.True(t, strings.Contains(payloadString, "\"Main_Page\",\"views\":10226718"))
```

Comment:
Can we improve DRY here by removing cut and paste?  Maybe candidate for table driven tests